app/src/main/java/it/ministerodellasalute/immuni/logic/exposure/repositories/ExposureReportingRepository.kt

```diff
@@ -74,4 +75,10 @@ class ExposureReportingRepository(
             storage[lastProcessedChunkKey] = value
         }
     }
+
+    var lastSuccessfulCheckDate: Date?
+        get() = storage[lastSuccessfulCheckDateKey]
+        set(value) {
+            storage[lastSuccessfulCheckDateKey] = value!!
```

Comment:
@sroddy 
It would be strange if setting nullable property failed. 
I suggest not to use force unwrap `value` but delete the key in storage if value is null.